tox.ini

```diff
@@ -5,7 +5,9 @@ indexserver =
         default = https://pypi.yelpcorp.com/simple/
 
 [testenv]
-deps = -rrequirements.txt
+deps =
+	-rrequirements.txt
+	-rtest-requirements.txt
```

Comment:
This is totally fine and a common pattern. Usually the file here is called requirements-dev.txt
https://github.com/search?q=requirements-dev.txt&type=Code&utf8=%E2%9C%93